google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py

```diff
@@ -33,8 +33,8 @@
 SUPPORTED_DATA_TYPE = frozenset([_IAM_POLICY, _RESOURCE])
 
 CAI_RESOURCE_TYPE_MAPPING = {
-    # TODO: Support non cai resource type by creating a fake cai resource type.
-    # e.g. 'lien' -> 'google.SOME_RESOURCE_TYPE.Lien'
+    'lien': 'cloudresourcemanager.googleapis.com/Lien',
```

Comment:
Please add a comment here that this is to map non-CAI resource types to a fake CAI resource type.  It might also be worth considering calling this `FAKE_CAI_RESOURCE_TYPE_MAPPING`.

Reply:
Fixed to MOCK_CAI_RESOURCE_TYPE_MAPPING